RecoVertex/V0Producer/src/V0Fitter.cc

```diff
@@ -116,12 +117,8 @@ void V0Fitter::fitAll(const edm::Event& iEvent, const edm::EventSetup& iSetup,
    // fill vectors of TransientTracks and TrackRefs after applying preselection cuts
    for (reco::TrackCollection::const_iterator iTk = theTrackCollection->begin(); iTk != theTrackCollection->end(); ++iTk) {
       const reco::Track* tmpTrack = &(*iTk);
-      double ipsigXY;
-      if (useVertex_) {
-         ipsigXY = std::abs(tmpTrack->dxy(referencePos)/tmpTrack->dxyError());
-      } else {
-         ipsigXY = std::abs(tmpTrack->dxy(*theBeamSpot)/tmpTrack->dxyError()); 
-      }
+      double ipsigXY = std::abs(tmpTrack->dxy(*theBeamSpot)/tmpTrack->dxyError());
+      if (useVertex_) ipsigXY = std::abs(tmpTrack->dxy(referencePos)/tmpTrack->dxyError());
```

Comment:
why computing twice ipsingXY in case of useVertex_?
was not the previous code more efficient?